pkg/utils/utils.go

```diff
@@ -0,0 +1,16 @@
+package utils
+
+import (
+	"errors"
+	"os"
+)
+
```

Comment:
Can you add the comment that was in the previous function here as well?

Reply:
Done :smile: